### PR TITLE
Add feature to hide the sidebar in the new Twitter UI

### LINKED ISSUE #183: New Twitter layout: "Trends for you" and "Who to follow" started showing up again.
This may be due to the new twitter layout. How can I contribute, I love this extension.

### PR DESCRIPTION
**Pull request by Tushkiz**
Inspect element shows that Twitter is using some kind of CSS class minification, but there is a way to select the whole sidebar and hide both at once. The downside of this is that it cannot be toggled individually.

I could hide it on the home page, but failing for `onRouteChange`

Fixes #183

### CODE REVIEW

**Comment by Tushkiz** [comment]
fixes #183

**Comment by Tushkiz** [comment]
Hi @filipekiss @jezdez
would like to hear any suggestion you may have for this

**Review by jorgegonzalez — APPROVED** [review]
Looks good to me

**Comment by sindresorhus** [comment]
So this targets the new layout? I'm not on that, so I cannot really test this.

**Comment by Tushkiz** [comment]
yes, this is for the new layout. Only works for the first load and comes back after route change

File: source/features/index.js [hunk 1]
```diff
@@ -73,17 +73,12 @@ export const features = {
 		label: 'Hide "Profile" Card',
 		runOnInit: true
 	},
-	hideHomeTrendsCard: {
```
[the remark this answers is not in the document]
Comment (sindresorhus) [reply source/features/index.js]: Yes

**Comment by sindresorhus** [comment]
I think it would make more sense to add this as a separate feature, and keep the old ones for now. The feature name should reflect that it's for the new design.

**Pull request merged**